From: Paul Holzinger <pholzing@redhat.com>
Date: Thu, 2 Apr 2026 18:31:41 +0200
Subject: fix handling of incorrect tcp packets

Ensure we correctly abort the tcp stream on errors and do not keep
retrying.

Signed-off-by: Paul Holzinger <pholzing@redhat.com>
---
 test/100-basic-name-resolution.bats | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/100-basic-name-resolution.bats b/test/100-basic-name-resolution.bats
index 43d40fa..9f3e3ab 100644
--- a/test/100-basic-name-resolution.bats
+++ b/test/100-basic-name-resolution.bats
@@ -371,7 +371,6 @@ function teardown() {
 	run_in_container_netns "$a1_pid" "dig" "+short" "second-server.test" "@$gw"
 	assert "$output" == "192.168.100.2" "should resolve using second DNS server after resolv.conf change"
 }
-
 @test "check for incorrect tcp packet" {
 	setup_dnsmasq
 
